client/src/js/services/VisitsService.js

```diff
@@ -0,0 +1,29 @@
+angular.module('bhima.services')
+.service('VisitService', VisitService);
+
+VisitService.$inject = [ '$http', 'util' ];
+
+function VisitService($http, util) {
+  var service = this;
+  var baseUrl = '/patients/';
+ 
+  service.read = read;
+  service.checkin = checkin; 
+  
+  function read(patientUuid, options) { 
```

Comment:
Should there be any error indications if the patientUuid doesn't exist instead of silently returning?